install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of the custom Broker to connect to
                         type: string
                     type: object
                   native:
                     properties:
                       name:
+                        description: Name of the native Broker to connect to
                         type: string
                       namespace:
+                        description: Namespace where the native Broker is deployed
                         type: string
                     type: object
                 type: object
               version:  
+                description: Version of the Broker to connect to
                 type: string
               size:
+                description: Desired size of the Broker
                 format: int32
                 type: integer
               watch-list:
                 type: object
                 properties:
                   data:
+                    description: Specifies the resource types and their associated whitelist and blacklist for watching
```

Comment:
"...for watching... ?"

Reply:
I have rectified the description accordingly.